pkg/app/pipedv1/plugin/kubernetes/livestate/plugin_test.go

```diff
@@ -0,0 +1,538 @@
+    app: test
+`),
+					makeTestDiffChange(t, `
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: test-ingress
+  namespace: default
+spec:
+  rules:
+  - host: old.example.com
+    http:
+      paths:
+      - path: /
+        pathType: Prefix
+        backend:
+          service:
+            name: test-service
+            port:
+              number: 80
+`, `
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: test-ingress
+  namespace: default
+spec:
+  rules:
+  - host: new.example.com
+    http:
+      paths:
+      - path: /
+        pathType: Prefix
+        backend:
+          service:
+            name: test-service
+            port:
+              number: 443
+`),
+				},
+			},
+			commitHash: "1234567",
+			want: sdk.ApplicationSyncState{
+				Status:      sdk.ApplicationSyncStateOutOfSync,
+				ShortReason: "There are 2 manifests not synced (0 adds, 0 deletes, 2 changes)",
+				Reason: `Diff between the defined state in Git at commit 1234567 and actual state in cluster:
+
+--- Actual   (LiveState)
++++ Expected (Git)
+
+# 1. name="test-service", kind="Service", namespace="default", apiGroup=""
+
+  spec:
+    ports:
+      -
+        #spec.ports.0.port
+-       port: 80
++       port: 443
+
+        #spec.ports.0.targetPort
+-       targetPort: 8080
++       targetPort: 8443
+
+
+# 2. name="test-ingress", kind="Ingress", namespace="default", apiGroup="networking.k8s.io"
+
+  spec:
+    rules:
+      -
+        #spec.rules.0.host
+-       host: old.example.com
++       host: new.example.com
+
+        http:
+          paths:
+            - backend:
+                service:
+                  port:
+                    #spec.rules.0.http.paths.0.backend.service.port.number
+-                   number: 80
++                   number: 443
+
+
+`,
+			},
+		},
+		{
+			name: "resource deletion and addition",
+			diffResult: &provider.DiffListResult{
+				Adds: []provider.Manifest{
+					makeTestManifest(t, `
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: test-pvc
+  namespace: default
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+`),
+				},
+				Deletes: []provider.Manifest{
+					makeTestManifest(t, `
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-pod
+  namespace: default
+spec:
+  containers:
+  - name: app
+    image: nginx:1.19
+`),
+				},
+			},
+			commitHash: "1234567",
+			want: sdk.ApplicationSyncState{
+				Status:      sdk.ApplicationSyncStateOutOfSync,
+				ShortReason: "There are 2 manifests not synced (1 adds, 1 deletes, 0 changes)",
+				Reason: `Diff between the defined state in Git at commit 1234567 and actual state in cluster:
+
+--- Actual   (LiveState)
++++ Expected (Git)
+
+- 1. name="test-pod", kind="Pod", namespace="default", apiGroup=""
+
++ 2. name="test-pvc", kind="PersistentVolumeClaim", namespace="default", apiGroup=""
+
+`,
+			},
+		},
+		{
+			name: "some resources are out of sync",
```

Comment:
```suggestion
			name: "config map data is masked",
```

Reply:
Oops, thank you

Reply:
Fixed on this commit https://github.com/pipe-cd/pipecd/pull/5676/commits/3278eba0b573822512664af5899c88cf6485ad43